### ISSUE TITLE
Explicit block parameters when calling new_stack

### ISSUE THREAD

**Issue by DivadNojnarg** [blockr-org/blockr#78]
Right now, when we call `new_stack`, its not clear what are the default block parameters or we might want to change them:

```r
new_stack(
  data_block,
  filter_block
)
```

We currently have to go back to the code base or use autocompletion to get an idea of what the block function is doing.

New:

```r
new_stack(
  data_block(data = mtcars, ...)
  filter_block(column = "mpg", value = 21.0 , ...),
 ...
)
```

**Comment by JohnCoene** [comment]
I think this suggestion is good and probably necessary.
E.g.: we will also need a way to restore a stack to specific settings in the future; this is an elegant way of going so.

**Comment by DivadNojnarg** [comment]
Note for future me:

using `as.list(substitute(...()))`  allows to get a list of calls such as:

```r
[[1]]
data_block(dat = as.environment("package:datasets"), selected = "iris")

[[2]]
select_block(selected = "Species")
```

**Comment by DivadNojnarg** [comment]
Fixed in 0.0.2 we can now do:

```r
library(blockr)
new_stack( # <1>
  data_block = new_dataset_block("penguins", "palmerpenguins"), # <2>
  filter_block = new_filter_block("sex", "female"), #<2>
  plot_block = new_ggplot_block("body_mass_g", "flipper_length_mm"), # <3>
  layer_block = new_geompoint_block("species", "species") # <4>
)
serve_stack(stack)
```
